app/main.py

```diff
@@ -189,5 +189,6 @@ def __init__(self, *args, **kwargs):
 init_session()
 middleware = [JSONTranslator(), DatabaseSessionManager(db_session), CORSMiddleware()]
 application = App(middleware=middleware)
```

Comment:
https://github.com/falconry/falcon/issues/1194
Setting A results in the same behavior as CORS Middleware,
I left it as it is, in case the allow origin is set dynamically in future enhancements.